tests/templates/kuttl/smoke/60-install-hive.yaml.j2

```diff
@@ -22,6 +22,12 @@ spec:
     config:
       logging:
         enableVectorAgent: {{ lookup('env', 'VECTOR_AGGREGATOR') | length > 0 }}
+      resources:
+        cpu:
+          max: '750m'
+          min: '250m'
+        memory:
+          limit: '1024Mi'
```

Comment:
WDYT of using the default resources, so that we see that some resources are set even than? Also to guarantee that the resources are sufficient to pass the smoke test